swarm/swarm.go

```diff
@@ -51,19 +57,29 @@ func run(cmd string) {
 }
 
 func getleaderHost() string {
```

Comment:
Right now we are using docker 1.5.0 and swarm required docker version >= 1.6.0. Also etcd2 doesn't support getting machines API.